bot/src/commands/wallet/SetCurrencyCodeCommand.ts

```diff
@@ -0,0 +1,67 @@
+import { Message } from "node-telegram-bot-api";
+import { SetWalletCurrencyCodeRequest } from "../../../../wallet/build/client";
+import { Bot } from "../../Bot";
+import { BotReplyToMessageIdHandler } from "../../handler";
+import { translationKeys } from "../../i18n";
+import { IBotCommand } from "../IBotCommand";
+import { getCurrencyButtons } from "../util/currencies";
+import getUserId from "../util/getUserId";
+
+export class SetCurrencyCodeCommand implements IBotCommand {
+  private bot: Bot;
+
+  constructor(bot: Bot) {
+    this.bot = bot;
+  }
+
+  async onReplyFromMessageID(
+    msg: Message,
+    handler: BotReplyToMessageIdHandler,
+    match?: RegExpMatchArray
+  ) {}
+
+  async onText(msg: Message) {
+    const currencyButtons = getCurrencyButtons();
+    try {
+      this.bot.reply(msg, translationKeys.command_text_set_currency, {
+        disable_web_page_preview: true,
+        reply_markup: {
+          resize_keyboard: true,
+          one_time_keyboard: true,
+          keyboard: [...currencyButtons],
```

Comment:
```suggestion
          keyboard: getCurrencyButtons(),
```

Same here. For the sake of simplicity, there's no need to clone the Array, and since we are not using the `const` in any other place...